air/src/air/trace_info.rs

```diff
@@ -200,11 +200,6 @@ impl TraceInfo {
     pub fn get_aux_segment_width(&self) -> usize {
         self.aux_segment_width
     }
-
-    /// Returns the number of random elements required by the auxiliary trace segment.
-    pub fn get_num_aux_segment_rand_elements(&self) -> usize {
-        self.num_aux_segment_rands
-    }
```

Comment:
Do we not need this any more? Seems a bit odd that we are removing the accessor but keeping the field (i.e., `self.num_aux_segment_rands).

Reply:
Oops, I forgot to remove the field. Fixed.

We don't need it anymore because the `TraceInfo` doesn't know anymore how many random elements will be generated (e.g. in the case of GKR, that number is dynamic)

Reply:
Note: I also removed `MAX_RAND_SEGMENT_ELEMENTS`, which I believe was only a `TraceInfo::to_elements()` limitation (it had to fit in a `u8`)

Reply:
I left a comment in the review below about this.